### PR TITLE
Add eslint-plugin-mozilla plugin

### PR DESCRIPTION
**Pull request by pdehaan**

### CODE REVIEW

File: src/settings/settings.js [hunk 1]
```diff
@@ -3,9 +3,11 @@ const themeLegend = document.getElementById('themeTitle');
 const defaultThemeLabel = document.getElementById('default_label');
 const darkThemeLabel = document.getElementById('dark_label');
 
+/* eslint-disable no-unsanitized/property */
```
Comment (pdehaan) [inline src/settings/settings.js]: I just disabled the rule for these 3 lines. Not sure if there is some "sanitization" module we wanted to use or something to make sure there aren't any potential XSS leaks from the l10n files.

**Comment by vladikoff** [comment]
@pdehaan we shall take a look at it once we tag the sync build, thanks for the pr!

**Comment by vladikoff** [comment]
@pdehaan Could you please rebase this?

**Comment by vladikoff** [comment]
@pdehaan could you please rebase this ?

**Comment by pdehaan** [comment]
@vladikoff Consider yourself rebased... I also bumped the **eslint-plugin-mozilla** and **eslint** dependency versions and ran `eslint src --fix` to auto-fix all the formatting errors.

I also noticed that indenting _may_ be a bit funky in some places since we don't seem to have any `indent` rules enabled in ESLint. I _could_ run it through prettier (`npm run format`), but not sure how noisy and disruptive that would be, so I'd save it for a future PR.

https://github.com/mozilla/notes/blob/03cd44a4ed31470d63d9b11f2675a2c7f9582556/package.json#L67

**Comment by Natim** [comment]
Thanks @pdehaan we will definitely add prettier while moving to a React.JS codebase.

**Comment by vladikoff** [comment]
@pdehaan thank you!

**Comment by mozfreddyb** [comment]
> [@vladikoff wrote](https://github.com/mozilla/notes/pull/478#issuecomment-349695378)
> we shall take a look at [sanitizing html] once we tag the sync build, thanks for the pr!

so, @vladikoff did you? :)
If not, I recommend looking at https://github.com/cure53/DOMPurify

**Comment by vladikoff** [comment]
@pdehaan any luck adding that ^ ?

**Pull request merged**